app/presenters/hyrax/menu_presenter.rb

```diff
@@ -0,0 +1,32 @@
+module Hyrax
+  # view-model for the admin menu
+  class MenuPresenter
+    def initialize(view_context)
+      @view_context = view_context
+    end
+
+    attr_reader :view_context
+
+    delegate :controller, :controller_name, :action_name, :content_tag,
+             :current_page?, :link_to, to: :view_context
+
+    # Returns true if the current controller happens to be one of the controllers that deals
+    # with workflow.  This is used to keep the parent section on the sidebar open.
+    def workflows_section?
+      controller.instance_of? Hyrax::Admin::WorkflowRolesController
+    end
+
+    def nav_link(href, **link_options)
+      content_tag(:li, class: ('active' if current_page?(href))) do
```

Comment:
Inline terinary operators for hash values strikes me as a bit stinky. It's hard to insert a debugger to see the results.

```ruby
content_tag_options = {}
content_tag_options[:class] = 'active' if current_page?(href)
content_tag(:li, content_tag_options) do
```

Optionally, I can get behind:

```ruby
content_tag_options = current_page?(href) ? { class: 'active' } : {}
content_tag(:li, content_tag_options) do
```